gef-ui/templates/partials/confirm-cover-start-date.njk

```diff
@@ -0,0 +1,51 @@
+{% extends "index.njk" %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% block pageTitle %}
+    Confirm cover start date for issued facilities
+{% endblock %}
+
+{% block content %}
+
+    {% if errors %}
+        {{
+            govukErrorSummary({
+            titleText: "There is a problem",
+            errorList: errors.errorSummary,
+            attributes: {
+                'data-cy': 'error-summary'
+                }
+            })
+        }}
+    {% endif %}
+
+    {% set hide_back = 1 %}
```

Comment:
can we change this to boolean? And move to the UI server so the template just receives `hideBack`?